packages/components/pie-input/src/index.ts

```diff
@@ -29,20 +31,38 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
     @property({ type: String })
     public value = '';
 
+    @property({ type: String })
+    public name = '';
+
+    protected firstUpdated (_changedProperties: PropertyValues<this>): void {
```

Comment:
I think its entirely possible. I think we should start by waiting to see what parts get repeated each time we add a new form control. Otherwise we risk creating a mixin that is too rigid in its assumptions about a component's implementation.

Once we see need for duplication, we can bring the logic into the mixin 😃